2024/src/pages/index.tsx

```diff
@@ -263,7 +266,7 @@ export default function IndexPage() {
       subTitle: t("callForSponsors"),
       available: site.siteMetadata.sponsorFormUrl,
       render: () =>
-        false ? (
+        !sponsorDateOverDue ? (
```

Comment:
Here I think it should be `site.siteMetadata.sponsorFormUrl` I believe.

Reply:
The event has three states:

1. There is no sponsor URL: It should show the "coming-soon" state.
2. There is a sponsor URL and it is active: It should show the Sponsor button
3. There is a sponsor URL but the `over-due-date` has passed: nothing should be shown.

This should do the trick:

<img width="492" alt="Screenshot 2024-10-31 at 17 36 31" src="https://github.com/user-attachments/assets/67fbdbe4-9cc9-43da-b63d-de3abac12c01">